protocol/x/prices/keeper/slinky_adapter_test.go

```diff
@@ -76,33 +75,6 @@ func TestGetPriceForCurrencyPair(t *testing.T) {
 	})
 	require.Error(t, err)
 }
-
-func TestBadMarketData(t *testing.T) {
-	ctx, keeper, _, _, mockTimeProvider, _ := keepertest.PricesKeepers(t)
-	mockTimeProvider.On("Now").Return(constants.TimeT)
-
-	_, err := keeper.CreateMarket(
-		ctx,
-		types.MarketParam{
-			Id:                 uint32(0),
-			Pair:               "00",
-			MinExchanges:       1,
-			MinPriceChangePpm:  1,
-			ExchangeConfigJson: "{}",
-		},
-		types.MarketPrice{})
-	require.NoError(t, err)
```

Comment:
With the requirement that a market can't be created if it doesn't already exist in marketmap, we can no longer create a market with malformed Pair